docs/appendices/release-notes/5.10.0.rst

```diff
@@ -105,3 +105,9 @@ Administration and Operations
 
 - Added node settings for the :ref:`jwt_defaults` allowing to provide global
   values for the :ref:`JWT properties <create-user-jwt>`.
+
+User Interface
+--------------
+
+- Updated to Admin UI 1.24.8, which is a maintenance build using an updated
+  set of dependencies.
```

Comment:
Seems like this update has no user visible change, such I'd leave an entry out.
```suggestion
```

Reply:
I've just removed that change log item. Thanks for the guidance.